magni_bringup/scripts/launch_core.py

```diff
@@ -107,42 +121,59 @@ def create_core_launch_file(
                 {
                     "camera_extrinsics_file": camera_extrinsics_file,
                     "lidar_extrinsics_file": lidar_extrinsics_file,
-                    "sonars_installed": sonars_installed,
+                    "sonars_installed": conf["sonars_installed"],
                     "oled_display": oled_display,
                     "controller_board_version": str(board_rev),
-                    "serial_port": str(motor_controller_params["serial_port"]),
-                    "serial_baud": str(motor_controller_params["serial_baud"]),
-                    "pid_proportional": str(
-                        motor_controller_params["pid_proportional"]
-                    ),
-                    "pid_integral": str(motor_controller_params["pid_integral"]),
-                    "pid_derivative": str(motor_controller_params["pid_derivative"]),
-                    "pid_denominator": str(motor_controller_params["pid_denominator"]),
-                    "pid_moving_buffer_size": str(
-                        motor_controller_params["pid_moving_buffer_size"]
-                    ),
-                    "pid_velocity": str(motor_controller_params["pid_velocity"]),
-                    "wheel_separation_multiplier": str(
-                        vc_params["wheel_separation_multiplier"]
-                    ),
-                    "wheel_radius_multiplier": str(
-                        vc_params["wheel_radius_multiplier"]
-                    ),
-                    "has_velocity_limits": str(vc_lin["has_velocity_limits"]),
-                    "has_velocity_limits": str(vc_lin["max_velocity"]),
-                    "has_acceleration_limits": str(vc_lin["has_acceleration_limits"]),
-                    "max_acceleration": str(vc_lin["max_acceleration"]),
-                    "has_velocity_limits": str(vc_ang["has_velocity_limits"]),
-                    "max_velocity": str(vc_ang["max_velocity"]),
-                    "has_acceleration_limits": str(vc_ang["has_acceleration_limits"]),
-                    "max_acceleration": str(vc_ang["max_acceleration"]),
+
```

Comment:
Interesting use of this 'mot_cont' to make the reading here easier.  Nice.  I think that was pre-existing so I assume it works.

Reply:
It would also be extremely nice if someone told me they have tested this on their actual robots and it works. I have tested on mine, but you never know.